go/pkg/sdk/optimism/sdk_optimism.go

```diff
@@ -268,6 +268,72 @@ func (sdk *SdkOptimism) GetSlippage(stable constants.Stablecoin, amount *big.Int
 	return percentageChange, expectedPriceFloat, executedPriceFloat, nil
 }
 
+func(sdk *SdkOptimism) createSwapData(callTo common.Address, approveTo common.Address, sendingAssetId common.Address, receivingAssetId common.Address, fromAmount *big.Int, fromStableTosDai bool) ([]optimismContracts.LibSwapSwapData, error) {
+	swapABI, err := abi.JSON(strings.NewReader(optimismContracts.IVelodromeMetaData.ABI))
+    if err != nil {
+        return nil, fmt.Errorf("failed to parse ABI: %w", err)
+    }
+
+	client, err := ethclient.Dial(sdk.Config.RpcEndpoint)
+	if err != nil {
+		return nil, fmt.Errorf("Error loading client: %w", err)
+	}
```

Comment:
this code should be indented. not a big deal here, but we'll add a go linter as pre-commit hook (and check in CI) to avoid that in the future